### PR TITLE
[Pelias] Fix NULL value for $bounds

### LINKED ISSUE #1043: [Pelias] Notice: Trying to access array offset on value of type null on php 7.4
Hi
Got error on **php 7.4** 
**Notice: Trying to access array offset on value of type null on php**

in vendor/willdurand/geocoder/Model/Address.php (line 258)

### PR DESCRIPTION
**Pull request by jbelien**
In `Address::createFromArray()` function, `$bound` needs to be an array !

Fix #1043

### CODE REVIEW

File: src/Provider/Pelias/Pelias.php [hunk 1]
```diff
@@ -166,7 +166,12 @@ protected function executeQuery(string $url): AddressCollection
                     'east' => $location['bbox'][0],
                 ];
             } else {
-                $bounds = null;
+                $bounds = [
```
Comment (atymic) [inline src/Provider/Pelias/Pelias.php]: maybe worth adding a test case to cover this bug?

Comment (jbelien) [reply src/Provider/Pelias/Pelias.php]: Good idea! 👍

Comment (jbelien) [reply src/Provider/Pelias/Pelias.php:169]: Test added but I don't have an API key to add the cached response (I requested one).

**Comment by jbelien** [comment]
@atymic Would you have time to review my last changes, so I can merge this ? Thanks

PS: I'll fix CI (failed) test in `master` (since it's not related to this PR) and Travis builds passed (not sure why it still shows as pending here)

**Pull request merged**